cli/generator-op-microservice/generators/microservice/templates/src/typedef.graphql

```diff
@@ -1,7 +1,7 @@
 type Query {
   # Query Definitions
   list: [<%= serviceClassName %>Type]
-  get(ID: ID!): <%= serviceClassName %>Type
+  get(_id: String!): <%= serviceClassName %>Type
```

Comment:
Avoid using `_` for arg names in graphql.
```suggestion
  get(id: String!): <%= serviceClassName %>Type
```

Reply:
Same for other queries and. mutations.

Reply:
Updated